### ISSUE TITLE
Row disappears on delete if it shouldn't

### ISSUE THREAD

**Issue by silver17** [AllenFang/react-bootstrap-table#1655]
In my table, I have defined deleting row like this:

```
 remote(obj) {
    obj.cellEdit = true;
    obj.insertRow = true;
    obj.dropRow = true;
    return obj;
  }

deleteRow(currentRows) {
    this.props.removeParameters(Entities.COUNTRIES, currentRows);
  }

 const options = {
      onDeleteRow: ::this.deleteRow,

```

I'm using reducers to handle state changes. If I try to delete some row which can't be deleted, I get an error from backend. But before getting that error, the row disappears for some reason from UI. And it's not coming from my reducer where I have defined what to do in case rowDelete action. I managed to track things to ReactUpdates.js under react-dom and this part makes the row gone 

`    ReactReconciler.performUpdateIfNecessary(component, transaction.reconcileTransaction, updateBatchNumber);`

But I have no idea why. Any ideas? 

PS. After the row is gone, I receive the error from backend and display the alert.

Tried with v3.4.4 and v4.0.5

**Comment by AllenFang** [comment]
@silver17 hmm.. it's a critical bug,,,, I'll fix it soon, thanks

**Comment by AllenFang** [comment]
@silver17 sorry for lately reply, this was fixed on `v4.1.0`

**Comment by silver17** [comment]
Thanks, seems to fixed.
